script/move_source_fille_utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+"""This utils is used in chromium tools/git/move_source_file.py."""
+
+import os
+
+BRAVE_DIR_NAME = 'brave'
+
+def is_in_brave_dir():
+  _, tail = os.path.split(os.getcwd())
+  return tail == BRAVE_DIR_NAME
+
+
+def to_src_relative_path(dir):
```

Comment:
we can shorten patched lines a little by accepting `*args` and returning values list
https://stackoverflow.com/questions/8381739/how-to-return-multiple-values-from-args

Reply:
Done